main.go

```diff
@@ -1,9 +1,15 @@
 package main
 
 import (
-	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
-
-	"github.com/terraform-providers/terraform-provider-random/internal/provider"
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-mux/tf5muxserver"
+	"github.com/hashicorp/terraform-plugin-mux/tf6to5server"
+	"github.com/terraform-providers/terraform-provider-random/internal/provider_fm"
```

Comment:
I think this shouldn't compile: from what I see `provider_fm` was just a temporary module that you have got rid of, now that the PR is ready.

Reply:
You are right sir. Fixed